src/robusta/integrations/webex/sender.py

```diff
@@ -0,0 +1,254 @@
+            body.append(container)
+
+    def _createMessageContentJSON(self, message_content, description):
+
+        message_content_container = {
+            "type": "Container",
+            "items": [
+                {"type": "TextBlock", "text": message_content, "wrap": "true"},
+                {"type": "TextBlock", "text": description, "wrap": "true"},
+            ],
+        }
+        return message_content_container
+
+    def _createAdaptiveCard(self, blocks):
+
+        # https://learn.microsoft.com/en-us/adaptive-cards/
+        # metadata for adaptive cards
+        adaptive_card = {
+            "type": CardTypes.ADAPTIVE_CARD.value,
+            "$schema": ADAPTIVE_CARD_SCHEMA,
+            "version": ADAPTIVE_CARD_VERSION,
+        }
+
+        # Creating a container from message_content and description of finding for adaptive card
+
+        adaptive_card["body"] = blocks
+
+        return adaptive_card
+
+    def _seperate_blocks(self, finding: Finding, platform_enabled: bool):
+        table_blocks: List[TableBlock] = []
+        file_blocks: List[FileBlock] = []
+        description = None
+
+        message_content = self._create_message_content(
+            finding, platform_enabled, self.cluster_name
+        )
+
+        blocks = [MarkdownBlock(text=f"*Source:* _{self.cluster_name}_\n\n")]
+
+        # Seperate blocks into *Other* Blocks, TableBlocks and FileBlocks
+        for enrichment in finding.enrichments:
+            blocks.extend(
+                [
+                    block
+                    for block in enrichment.blocks
+                    if self.__is_webex_text_block(block)
+                ]
+            )
+            table_blocks.extend(
+                [block for block in enrichment.blocks if isinstance(block, TableBlock)]
+            )
+            file_blocks.extend(
+                [block for block in enrichment.blocks if isinstance(block, FileBlock)]
+            )
+
+        # first add finding description block
+        if finding.description:
+            if table_blocks:
+                description = finding.description
+            else:
+                blocks.append(MarkdownBlock(finding.description))
+
+        # Convert *Other* blocks to markdown
+        for block in blocks:
+            block_text = Transformer.to_standard_markdown([block])
+            if (
+                len(block_text) + len(message_content) >= MAX_BLOCK_CHARS
+            ):  # webex message size limit
+                break
+            message_content += block_text + "\n"
+
+        return message_content, table_blocks, file_blocks, description
+
+    def _send_files(self, files: List[FileBlock]):
+        # Webex allows for only one file attachment per message
+        # This function sends the files individually to webex
+        for blocks in files:
+            file_type = (
+                FileTypes.PHOTO if is_image(blocks.filename) else FileTypes.DOCUMENT
```

Comment:
It seems this enum is reduandent here, you could just check is_image

Reply:
Updated. Let me know if you want any more changes.